packages/reg-notify-github-with-api-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "reg-notify-github-with-api-plugin",
-  "version": "0.10.3",
+  "version": "0.11.0",
```

Comment:
version is managed by lerna cmd. Pls revert this line.

Reply:
Done.